EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,1118 @@
+---
+EIP: TBD
+
+Title: Royalty Bearing NFTs
+
+Description: This proposal outlines the structure of an extension of the ERC721 standard for NFTs to correctly define, process, and pay (hierarchical) onchain royalties from NFT sales, and goes beyond [EIP-2981](https://eips.ethereum.org/EIPS/eip-2981).
```

Comment:
```suggestion
description: This proposal outlines the structure of an extension of the ERC721 standard for NFTs to correctly define, process, and pay (hierarchical) onchain royalties from NFT sales, and goes beyond [EIP-2981](https://eips.ethereum.org/EIPS/eip-2981).
```

Reply:
This is too long, and shouldn't link to anything.  `ERC721` should be `EIP-721` as well.